app/routes/admin/content/system-images.js

```diff
@@ -5,5 +5,917 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('System Images');
+  },
+
+  model() {
+    console.log(this.get('imageTopic'));
+    return [
+      {
```

Comment:
Don't put everything in the model ! Just put one or two. Like I said before, server response may not have all images. It'll have only those that have a custom format set.